apps/test/unit/tutorialExplorer/imageTest.jsx

```diff
@@ -1,24 +1,30 @@
 import React from 'react';
 import {shallow} from 'enzyme';
-import {expect} from '../../util/configuredChai';
+import {assert} from '../../util/reconfiguredChai';
 import Image from '@cdo/apps/tutorialExplorer/image';
 
 describe('Image', () => {
   it('renders with minimum opacity at first', () => {
-    const wrapper = shallow(<Image style={{}} />);
-    expect(wrapper).to.containMatchingElement(<img style={{opacity: 0.1}} />);
+    const wrapper = shallow(<Image style={{}} />, {
+      disableLifecycleMethods: true
+    });
```

Comment:
[Shallow rendering now calls lifecycle methods like `componentDidMount` by default](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#lifecycle-methods) which is a problem for this test because [image.jsx uses `componentDidMount`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/tutorialExplorer/image.jsx#L21) and the test tries to ignore that.

Doing the easy fix for now by disabling these lifecycle methods.  Maybe worth revisiting in the future?